### DISCUSSION TITLE
关于第三周要提交的ppt的要求

### DISCUSSION THREAD

**Discussion by eyja42** [liusy58/NJU_AdvancedProgramming_SP22#69] [Q&A] [answered]
之前样例里给的那个示范ppt应该是整个项目做完之后的展示吧，在现在这个大部分人都甚至没有新建文件夹的情况下是不可能做到那种细致程度的吧
所以说具体在ppt里要包含哪些内容呢？

**Comment by RicoloveFeng** [comment, marked as answer]
展示你的设计，比如数据结构，函数功能，调用关系，proj的需求是怎么解决的
